os/kernel/task/task_prctl.c

```diff
@@ -209,7 +209,17 @@ int prctl(int option, ...)
 		ret = messaging_remove_list(port_name);
 		return ret;
 	}
-#endif
+	break;
+#else /* CONFIG_MESSAGING_IPC */
```

Comment:
This new types are under `#if CONFIG_TASK_NAME_SIZE > 0`. You should move line 223~227.

Reply:
Done.